staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1alpha1/api.pb.go

```diff
@@ -324,7 +326,7 @@ func NewRegistrationClient(cc *grpc.ClientConn) RegistrationClient {
 
 func (c *registrationClient) GetInfo(ctx context.Context, in *InfoRequest, opts ...grpc.CallOption) (*PluginInfo, error) {
 	out := new(PluginInfo)
-	err := c.cc.Invoke(ctx, "/pluginregistration.Registration/GetInfo", in, out, opts...)
+	err := c.cc.Invoke(ctx, "/v1alpha1.Registration/GetInfo", in, out, opts...)
```

Comment:
same question about whether this means the endpoint or wire format/content is changing